### PR TITLE
Mar 731 refactor signup page

### PR DESCRIPTION
**Pull request by marcoalfonso**

### CODE REVIEW

File: app/components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.join {
```
Comment (matt-sm) [inline app/components/Header/Header.css]: why does this component have .css where others have .scss?

Comment (marcoalfonso) [inline app/components/Header/Header.css]: Ah removed, was waiting for the header work to start using scss on header.

**Comment by matt-sm** [comment]
General comment - should we do some unit tests?

**Comment by marcoalfonso** [comment]
Yeah unit tests for sure, but maybe once the whole page is completed

**Pull request merged**